### DISCUSSION TITLE
android14-kernel-6.1 wifi问题

### DISCUSSION THREAD

**Discussion by jayyhancock** [tiann/KernelSU#1201] [General]
在小米14上使用谷歌官方的内核内核WIFI不能打开，数据也失效了
通过logcat取得
01-07 18:19:13.263  1567  1589 E android.hardware.wifi-service: Failed to access driver state control param No such file or directory, 2: No such file or directory
01-07 18:19:13.264  1567  1589 E android.hardware.wifi-service: Failed to load WiFi driver
01-07 18:19:13.264  1567  1589 E android.hardware.wifi-service: Failed to initialize firmware mode controller
01-07 18:19:13.265  1567  1589 E android.hardware.wifi-service: Wifi HAL start failed
01-07 18:19:13.265  1567  1589 E IPCThreadState: binder thread pool (1 threads) starved for 1004 ms
01-07 18:19:13.265  2978  3899 E WifiHalAidlImpl: start failed with service-specific exception: android.os.ServiceSpecificException:  (code 9)
01-07 18:19:13.266  2978  3899 E HalDevMgr: Cannot start IWifi. Status: 9
01-07 18:19:13.266  2978  3899 E WifiVendorHal: Failed to start vendor HAL
01-07 18:19:13.266  2978  3899 E WifiNative: Failed to start vendor HAL
01-07 18:19:13.266  2978  3899 E WifiNative: Failed to start Hal
01-07 18:19:13.266  2978  3899 E WifiClientModeManager[190310:unknown]: Failed to create ClientInterface. Sit in Idle
01-07 18:19:13.266  2978  3899 E WifiActiveModeWarden: ClientModeManager start failed!ConcreteClientModeManager{id=190310 iface=null role=null}
01-07 18:19:13.267  2978  3899 D WifiScanRequestProxy: Sending scan available broadcast: false
01-07 18:19:13.267  2978  3903 I WifiScanningService: Received a request to disable scanning, UID = 1000
01-07 18:19:13.267  2978  3903 I WifiScanningService: wifi driver unloaded
01-07 18:19:13.268  2978  3899 I WifiScanRequestProxy: Scanning is disabled
01-07 18:19:13.268  2978  3899 I WifiCountryCode: No active mode, call onDriverCountryCodeChanged with Null
01-07 18:19:13.269  2978  3899 E HalDevMgr: IWifiEventCallback.onFailure. Status: 9
01-07 18:19:13.269  2978  3899 E WifiHalAidlImpl: Unable to call isStarted because IWifi is null.
01-07 18:19:13.269  2978  3899 I WifiNative: Vendor HAL died. Cleaning up internal state.
01-07 18:19:13.269  2978  3899 D WifiController: STA disabled, return to DisabledState.
01-07 18:19:13.269  2978  3899 D WifiController: EnabledState.exit()
01-07 18:19:13.270  6598  6598 I OsAgent : onReceive: android.net.wifi.action.WIFI_SCAN_AVAILABILITY_CHANGED
01-07 18:19:13.271  2978  3899 D WifiController: DisabledState.enter()
01-07 18:19:13.271  2978  3899 E WifiActiveModeWarden: One of the native daemons died. Triggering recovery
01-07 18:19:13.271  6598  8612 I OsAgent : updateWiFiScanAvailState, scanAvailable: false
01-07 18:19:13.272  2978  3899 E WifiSelfRecovery: Triggering recovery for reason: WifiNative Failure
01-07 18:19:13.273  2978  3899 E WifiSelfRecovery: Already restarted wifi 2 times in last 1 hour. Disabling wifi
01-07 18:19:13.273  2978  3899 D WifiController: Recovery has been throttled, disable wifi
01-07 18:19:13.273  2978  3899 D WifiActiveModeWarden: Shutting down all mode managers

**Comment by androiddisk** [comment]
同问

[a comment by an automated account is not shown]
